Vizzuality/heco-invest
# FE - Small impact visualisation component (search results)

## Description

This PR adds an impact chart to the project cards.  

Instead of creating yet another component for this, I realised we could more easily add a `compactMode` to the existing `ImpactChart` component. 

**In this PR:**  
- Added a `compactMode` prop to the `ImpactChart`  
  - Compact mode displays only the shape, respecting category colours  
  - Added a placeholder matching the designs for when there's no impact data available  
- `ImpactChart` background colour fix  
  _Broken by myself in #177_
  - Re-added a background colour to the chart   
  - Improved the styling (two circles with different transparencies) to better match the designs  
- Updated Storybook  
  - With 4 stories, 2 for the normal ImpactChart (w/data + placeholder) and another 2 for compact mode (w/data + placeholder)  
- Removed impact chart placeholder image  
  _previously used in the project cards_

## Testing instructions

Verify that:  
- The display of the charts is correct on Storybook  
- Project cards in the project search view display the placeholders (we have no impact data passed yet)  
- There are no visual changes on the project page when it comes to the ImpactChart

## Tracking

[LET-422](https://vizzuality.atlassian.net/browse/LET-422)

## Screenshots 

NORMAL MODE

**With impact data:** 
<img width="480" alt="Screenshot 2022-05-26 at 13 11 25" src="https://user-images.githubusercontent.com/6273795/170486596-88afac0c-ca84-4438-b8df-5fd61ba39492.png">

**Placeholder:** 
<img width="484" alt="Screenshot 2022-05-26 at 13 11 44" src="https://user-images.githubusercontent.com/6273795/170486609-f9e39c12-c997-4f29-8268-68560d655bca.png">

COMPACT MODE

**With impact data:** 
<img width="608" alt="Screenshot 2022-05-26 at 13 12 48" src="https://user-images.githubusercontent.com/6273795/170486632-3efef37b-2168-44e7-b33f-70221da205a4.png">

**Placeholder:** 
<img width="613" alt="Screenshot 2022-05-26 at 13 20 57" src="https://user-images.githubusercontent.com/6273795/170486680-37b8324c-d9ce-4619-bbfb-7389a5ad72ae.png">
